src/judgeval/common/tracer.py

```diff
@@ -816,14 +816,15 @@ def wrap(client: Any) -> Any:
     Wraps an API client to add tracing capabilities.
     Supports OpenAI, Together, and Anthropic clients.
     """
-    tracer = Tracer._instance  # Get the global tracer instance
-    
     # Get the appropriate configuration for this client type
     span_name, original_create = _get_client_config(client)
     
     def traced_create(*args, **kwargs):
-        # Skip tracing if no active trace
-        if not (tracer and tracer._current_trace):
+        # Get the current tracer instance (might be created after client was wrapped)
+        tracer = Tracer._instance
```

Comment:
Does this fix work because we fetch the Tracer._instance when we call `traced_create()` rather than outside the function (which is called when we run `wrap()`, which takes place before we call `Tracer()`)?